webapp/src/components/Wallet/LightningWalletCard.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { WalletCard } from './WalletCard'
+import { useTranslation } from 'react-i18next'
+import { BalanceProgressBar } from './BalanceProgressBar'
+
+export const LightningWalletCard: React.FC<LightningWalletBalance> = ({
+  availableBalance,
+  unconfirmedBalance,
+  autoTransferLimit
+}) => {
+  const { t } = useTranslation('common')
+
+  return (
+    <WalletCard standardSize={4}>
```

Comment:
To jest prawie identyczne do ChannelsBalanceCard
Zrobmy komponent gdzie argumentami beda rzeczy ktore sie roznia, czyli np. currentValue, maxValue, title, color